corehq/apps/api/tests/audit_resources.py

```diff
@@ -0,0 +1,80 @@
+from corehq.apps.api.resources import v0_5
+from corehq.apps.auditcare.models import NavigationEventAudit
+from django.test import TestCase
+
+from datetime import datetime, timedelta, date
+from zoneinfo import ZoneInfo
+
+
+class DomainNavigationEventAudits:
+    def __init__(self, domain: str, project_time_zone: ZoneInfo):
+        self.domain = domain
+        self.timezone = project_time_zone
+        self.logs = {}
+
+    def add_log(self, user: str, date_time: datetime):
+        self.logs.setdefault(user, set()).add(date_time)
```

Comment:
I made it a set instead of a list just because order didn't matter so I didn't need a list. 
I do see I call `cls.domain1_audits.add_log(cls.username1, single_datetime)` twice, I can't recall why and you're right there's no reason.